WPMedia/arc-themes-components
# Updating 2.1.3 with changes from 2.1.2

## Ticket

- [THEMES-1600](https://arcpublishing.atlassian.net/browse/THEMES-1600)

## Description

Updates 2.1.3 with hotfixes applied to 2.1.2

## Test Steps

1. Checkout branch - `git checkout THEMES-1600`
2. Update dependencies - `npm i`
3. Run `npm run test` and `npm run lint`
4. Tests should pass and there should be no errors.

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._


## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1600]: https://arcpublishing.atlassian.net/browse/THEMES-1600?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
